libkbfs/block_retrieval_queue.go

```diff
@@ -117,14 +121,16 @@ type blockRetrievalQueue struct {
 	prefetcher Prefetcher
 }
 
-var _ blockRetriever = (*blockRetrievalQueue)(nil)
+var _ BlockRetriever = (*blockRetrievalQueue)(nil)
 
 // newBlockRetrievalQueue creates a new block retrieval queue. The numWorkers
 // parameter determines how many workers can concurrently call Work (more than
 // numWorkers will block).
-func newBlockRetrievalQueue(numWorkers int, config blockRetrievalConfig) *blockRetrievalQueue {
+func newBlockRetrievalQueue(numWorkers int,
+	config blockRetrievalConfig) *blockRetrievalQueue {
 	q := &blockRetrievalQueue{
 		config:      config,
+		log:         config.MakeLogger("BRQ"),
```

Comment:
You don't need a prefix for the logger if there's only one instance of the queue.  The printed filename is enough in my opinion.